src/main/java/it/pagopa/pn/bff/mappers/InfoPaMapper.java

```diff
@@ -0,0 +1,38 @@
+package it.pagopa.pn.bff.mappers;
```

Comment:
Maybe it will be needed split the mapper into two mappers, one for BffInstitutionList and one for BffInstitutionProductList. They are two separated entities and it is correct to have two separate mappers